### ISSUE TITLE
Physical bone joint bug

### ISSUE THREAD

**Issue by MadaraZ** [godotengine/godot#26060]
**Godot version:**
3.1.beta.custom_build. 1174d10e3c8f84e15dd6c5e795b5a6679c64c59c

**OS/device including version:**
Ubuntu 16.04

**Issue description:**
https://godotengine.org/article/godot-ragdoll-system

click bone joint doesn’t work :(

Error ===>
```
modules/bullet/rigid_body_bullet.cpp:463 - A body with a joints is destroyed. Please check the implementation in order to destroy the joint before the body.

scene/3d/skeleton.cpp:627 - Condition ' bones[p_bone].physical_bone ' is true.
core/image.cpp:961 - Condition ' p_width <= 0 ' is true.

modules/bullet/bullet_physics_server.cpp:1224 - Condition ' body_A == body_B ' is true. returned: RID()
modules/bullet/bullet_physics_server.cpp:1234 - Condition ' !joint ' is true.

modules/bullet/bullet_physics_server.cpp:1292 - Condition ' body_A == body_B ' is true. returned: RID()
modules/bullet/bullet_physics_server.cpp:1338 - Condition ' !joint ' is true.
modules/bullet/bullet_physics_server.cpp:1322 - Condition ' !joint ' is true.
modules/bullet/bullet_physics_server.cpp:1460 - Condition ' !joint ' is true.

modules/bullet/bullet_physics_server.cpp:1364 - Condition ' body_A == body_B ' is true. returned: RID()
modules/bullet/bullet_physics_server.cpp:1444 - Condition ' !joint ' is true.

modules/bullet/bullet_physics_server.cpp:1374 - Condition ' !joint ' is true
./core/rid.h:153 - Condition ' !p_rid.is_valid() ' is true. returned: __null
```

**Steps to reproduce:**
![0](https://user-images.githubusercontent.com/17591539/53029135-4b44d480-3479-11e9-9597-4d5e473c5935.gif)

![1](https://user-images.githubusercontent.com/17591539/53029161-526be280-3479-11e9-91c5-c86a15d08492.gif)

joint_type Hingejoint (radius curves bad) ===>

![4](https://user-images.githubusercontent.com/17591539/53029228-716a7480-3479-11e9-9238-392b6881c826.png)

**Minimal reproduction project:**
https://drive.google.com/file/d/12RED80XoMKyM5b1woTieZGW9irYXuNKs (35mb)

**Comment by AndreaCatania** [comment]
Is it possible for you test if this bug still happen with: #28909

**Comment by AndreaCatania** [comment]
Fixed by #36008

![Screenshot from 2020-02-09 09-46-38](https://user-images.githubusercontent.com/8342599/74099180-1ebe1880-4b21-11ea-91db-ec6aa081bde4.png)
